bears/haskell/GhcModBear.py

```diff
@@ -15,7 +17,8 @@ class GhcModBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {CabalRequirement(package='ghc-mod', version='5.6.0')}
+    REQUIREMENTS = {CabalRequirement(package='ghc-mod', version='5.6.0'),
```

Comment:
You can now use [AnyOneOfRequirements](https://gitlab.com/coala/package_manager/blob/master/dependency_management/requirements/AnyOneOfRequirements.py) to wrap these two requirements in an OR statement.